### PR TITLE
Nullability

### PR DESCRIPTION
**Pull request by HurricanKai**
Also contains CLS compliance, but that's a tiny code change
See also #39

### CODE REVIEW

**Comment by HurricanKai** [comment]
Please review @Perksey 😄

File: src/Maths/Silk.NET.Maths/ClsCompliance.cs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿using System;
+
+[assembly: CLSCompliant(true)]
```
Comment (Perksey) [inline src/Maths/Silk.NET.Maths/ClsCompliance.cs:3]: Could this instead be defined using the csproj? It seems weird to have assembly attributes defined in a random file wheras the others are autogenerated using the csproj.

```xml
<ItemGroup>
    <AssemblyAttribute Include="System.CLSCompliant">
      <_Parameter1>true</_Parameter1>
    </AssemblyAttribute>
</ItemGroup>
```

Comment (HurricanKai) [reply src/Maths/Silk.NET.Maths/ClsCompliance.cs:3]: Didn't find how to do that, will apply!

Comment (HurricanKai) [reply src/Maths/Silk.NET.Maths/ClsCompliance.cs]: Well apparently
```cs
<ItemGroup>
    <AssemblyAttribute Include="System.CLSCompliant">
      <_Parameter1>true</_Parameter1>
    </AssemblyAttribute>
</ItemGroup>
```
will only function for `string`s, generating something like: `[assembly: System.CLSCompliant("true")]`
but I've renamed to AssemblyInfo.cs

Comment (Perksey) [reply src/Maths/Silk.NET.Maths/ClsCompliance.cs]: Huh that's a little annoying, all good.

**Pull request merged**